src/components/select/select.component.spec.ts

```diff
@@ -229,6 +231,25 @@ describe('Select Component', () => {
         expect(component.value).toBe('One');
     });
 
+    it('should not open dropdown when tabbing past select', (done) => {
+
+        Promise.resolve(dispatchTabs()).then(() => {
```

Comment:
Can you verify that this test is working as expected? If I revert your fix the test still passes, whereas this test should fail without your fix and pass after your fix